stock_picking_return_refund_option/hooks.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import SUPERUSER_ID, api, tools
+
+
+def pre_init_hook(cr):
```

Comment:
Please don't do that.

As already said, you must respect dependencies.

If your code depends on purchase_stock and sale_stock, put them in depends key in manifest. IMHO, splitting in more specialized modules is the best option.

Reply:
https://github.com/Tecnativa/account-invoicing/pull/3